lib/hammer_cli_foreman/smart_variable.rb

```diff
@@ -84,27 +93,32 @@ class CreateCommand < HammerCLIForeman::CreateCommand
       success_message _("Smart variable [%{variable}] created")
       failure_message _("Could not create the smart variable")
 
-      build_options :without => [:variable_type, :validator_type]
+      build_options :without => [:variable_type, :validator_type, :override_value_order]
 
       option "--variable-type", "VARIABLE_TYPE", _("Type of the variable."),
         :format => HammerCLI::Options::Normalizers::Enum.new(
             ['string', 'boolean', 'integer', 'real', 'array', 'hash', 'yaml', 'json'])
       option "--validator-type", "VALIDATOR_TYPE", _("Type of the validator."),
         :format => HammerCLI::Options::Normalizers::Enum.new(['regexp', 'list', ''])
+      option "--override-value-order", "OVERRIDE_VALUE_ORDER", _("The order in which values are resolved"),
+            :format => HammerCLI::Options::Normalizers::List.new
```

Comment:
Aren't we missing `include SmartVariableUpdateCreateCommons` for the create command here?